examples/custom-environment/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "custom-environment"
+version = "4.0.0-rc"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+publish = false
+
+[dependencies]
+ink = { path = "../../crates/ink", default-features = false }
+
+scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
+scale-info = { version = "2.3", default-features = false, features = ["derive"], optional = true }
+sp-io = { version = "12.0.0", default-features = false, features = ["disable_panic_handler", "disable_oom", "disable_allocator"] }
+sp-runtime = { version = "12.0.0", default-features = false }
+
+[dev-dependencies]
+ink_e2e = { path = "../../crates/e2e" }
+
+[lib]
```

Comment:
We've cleaned up some of the fields in `cargo-contract` recently. Can you remove `name` and `crate-type`? You'll probably need to do this in your other PR too

Reply:
:white_check_mark: :white_check_mark: